src/Forms/__tests__/TextInput.test.tsx

```diff
@@ -24,6 +24,7 @@ describe('Text input', function () {
   let getByRole: BoundFunction<GetByRole>;
```

Comment:
Doesn't look like there's any tests for the `disabled` prop in here...

Reply:
I had them in the test file - are they not showing up? 
![image](https://user-images.githubusercontent.com/29589689/99422961-0fe56d00-28ce-11eb-929d-e8a854a0a356.png)

Reply:
They weren't, no.
![image](https://user-images.githubusercontent.com/50675045/99428924-496da680-28d5-11eb-9d57-e9ccbd0dba65.png)


I think it was because you didn't add this in _this_ PR GitHub just wasn't showing them. But I do see them now.